qiskit/optimization/algorithms/minimum_eigen_optimizer.py

```diff
@@ -244,30 +255,29 @@ def eigenvector_to_solutions(eigenvector: Union[dict, np.ndarray, StateFn],
     return solutions
 
 
-def eval_operator_at_bitstring(operator: Union[WeightedPauliOperator, MatrixOperator],
-                               bitstr: str) -> float:
+def eval_operator_at_bitstring(operator: Union[OperatorBase, LegacyBaseOperator], bitstr: str,
```

Comment:
Is this method used anymore? It certainly is not covered by any unit tests now. The only two places I saw it called were in min_eig_optimizer, which no longer does so.

Reply:
Oh yes, it's not needed anymore, thanks! Removed in a07aea4.